license-classifications.yml

```diff
@@ -2416,7 +2416,7 @@ categorizations:
     categories:
       - "property:unchecked"
 
-  - id: "Public-domain-ref "
```

Comment:
Unrelated to this very change, but this should probably be named "LicenseRef-Public-Domain" instead, or with "namespaces" like "LicenseRef-DoubleOpen-Public-Domain" or "[LicenseRef-scancode-public-domain](https://scancode-licensedb.aboutcode.org/public-domain.html)".